tasks.py

```diff
@@ -1042,7 +1042,7 @@ def ci_tests(  # noqa: C901 - too complex (9)
     pytest_options = [f"--durations={slowest}", "-rEf"]
 
     if xdist:
-        pytest_options.append("-n auto")
+        pytest_options.append("-n 4")
```

Comment:
```suggestion
        pytest_options.append("-n logical")
```
I would try the `logical` param to use the number of "logical" cores instead of physical ones. 
https://pytest-xdist.readthedocs.io/en/stable/distribution.html